### PR TITLE
Adds an example using the different minimizers

### PR DESCRIPTION
**Pull request by StephenNneji**
Removes examples/originalDSPCBilayer 
Renames 'originalDSPCBilayerStructInput.mat' and 'originalDSPCBilayerProjectClass.mat' to remove 'original' prefix
Adds controlAlgorithms.m to example

### CODE REVIEW

File: tests/testExamples.m [hunk 1]
```diff
@@ -11,66 +12,88 @@
             'DSPCMonolayerStandardLayers',...
             'DSPCMonolayerTwoContrasts',...
             'orsoDSPCCustomLayers',...
-            'projectConversion'};
+            'projectConversion',...
+            'DSPCMonolayerCustomLayers',...
+            'controlAlgorithms'};
```
Comment (DrPaulSharp) [inline tests/testExamples.m:17]: The "controlAlgorithms" test adds considerably to the runtime and processing requirements for the test suite. Does it need to be run in full every time?

File: examples/controlAlgorithms.m [hunk 2]
```diff
@@ -11,32 +11,32 @@
...
+% plotRefSLD(problem2, result2);
+% 
+% figure(3); clf
+% plotRefSLD(problem3, result3);
+% 
+% figure(4); clf
+% plotRefSLD(problem4, result4);
+% 
+% figure(5); clf
+% plotRefSLD(problem5, result5);
```
Comment (DrPaulSharp) [inline examples/controlAlgorithms.m:42]: Am I right in thinking that RAT is run twice here, but only one set of results is plotted?

**Review by DrPaulSharp — APPROVED** [review]
Might be worth thinking about how many iterations of simplex are required to satisfy what needs to be tested here - there's no point in spinning the algorithm's wheels.

File: examples/controlAlgorithms.m (hunk 2 above)
Comment (StephenNneji) [reply examples/controlAlgorithms.m:42]: Missed that line

**Comment by StephenNneji** [comment]
> Might be worth thinking about how many iterations of simplex are required to satisfy what needs to be tested here - there's no point in spinning the algorithm's wheels.

Since it converges after a single iteration, I can set this to 1 maybe

**Comment by DrPaulSharp** [comment]
> > Might be worth thinking about how many iterations of simplex are required to satisfy what needs to be tested here - there's no point in spinning the algorithm's wheels.
> 
> Since it converges after a single iteration, I can set this to 1 maybe

Yeah, that's what I was thinking

**Pull request merged**